configs/nosite-x86_64/variables.yaml

```diff
@@ -5,7 +5,8 @@
 
 variables:
   batch_time: ''
-  mpi_command: 'mpirun -n {n_nodes} -c {n_ranks} --oversubscribe'
+  mpi_command: 'mpirun'
```

Comment:
Notes:

* With this PR (and if it is accepted, going forward), it should be left to `generate-batch` to insert options like `-n`
* For `mpirun` specifically, `-n` and `-c` are the same option (it differs from `srun` in that sense)